helix-contract/contracts/mapping-token/v2/Backing.sol

```diff
@@ -7,6 +7,14 @@ import "./AccessController.sol";
 contract Backing is AccessController, Initializable {
     address public messageHandle;
```

Comment:
what it message handle?

Reply:
It's the messageEndpoint we have discussed.